File: backend/GestionSetlistApp/Models/Setlist.cs
namespace GestionSetlistApp.Models
{
    public class Setlist
    {
        public int SetlistId { get; set; }
        public required string NomSetlist { get; set; }
        public int DureeSetlist { get; set; }
        public ICollection<Evenement> Evenements { get; set; } = [];
        public ICollection<MorceauSetlist> MorceauSetlists { get; set; } = [];
        public ICollection<MembreSetlist> MembreSetlist { get; set; } = [];

    }
}

File: frontend/gestion-setlist/src/app/models/setlist.ts
import { Evenement } from "./evenement";
import { MembreSetlist } from "./membreSetlist";
import { MorceauSetlist } from "./morceauSetlist";

export interface Setlist {
    setlistId: number;
    nomSetlist: string;
    dureeSetlist: number;
    evenements: Evenement[];
    morceauxSetlists: MorceauSetlist[];
    membres: MembreSetlist[];
}
